src/azure-cli/azure/cli/command_modules/rdbms/_params.py

```diff
@@ -374,7 +378,7 @@ def _flexible_server_params(command_group):
                        help='The start IP address of the firewall rule. Must be IPv4 format. Use value \'0.0.0.0\' to represent all Azure-internal IP addresses. ')
 
         with self.argument_context('{} flexible-server firewall-rule delete'.format(command_group)) as c:
-            c.argument('prompt', options_list=['--prompt'], help='Turn confirmation prompt on/off. If off, the rule will be deleted without confirmation')
+            c.argument('yes', options_list=['--yes', '-y'], action='store_true', help='Do not prompt for confirmation.')
```

Comment:
same

Reply:
we wanted to deleting align confirmation parameters same as deleting commands of other resources

Reply:
See above (and PR description udpates)